app/src/main/java/org/oppia/app/options/ReadingTextSizeActivity.kt

```diff
@@ -0,0 +1,63 @@
+package org.oppia.app.options
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import org.oppia.app.activity.InjectableAppCompatActivity
+import javax.inject.Inject
+
+/** The activity to change the Text size of the Story content in the app. */
```

Comment:
Update to `/** The activity to change the text size of the reading content in the app. */`